mobile/scripts/fetch_release.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+DIR="$(dirname "$(stat -f "$0")")"
+IOS_DIR="${DIR}/../ios"
+ANDROID_DIR="${DIR}/../android"
+
+# fetch iOS framework
+mkdir -p ${IOS_DIR}
+wget https://github.com/textileio/textile-go/releases/download/v0.1.4/textile-go_v0.1.4_ios-framework.tar.gz
```

Comment:
It's fine all these version numbers are hard coded for now?

Reply:
Seems like the version string could be passed as an arg from package.json to this script

Reply:
it's in a half-baked state now where this package version doesn't match the go release version (cause i had already published 0.1.4)... so, next release can clean all that up... ideally, in CI we'd bump both versions together, then auto-publish this... something like pass version to script, like you suggest. fine with it like this until next time around